scrapy_poet/utils.py

```diff
@@ -99,19 +76,9 @@ def is_provider_using_response(provider):
     """Check whether injectable provider makes use of a valid Response."""
```

Comment:
`discover_callback_providers` method was updated to make use of `andi.plan` again.

Reply:
`is_provider_using_response ` method was updated to make use of andi.plan. I had to move the `SCRAPY_PROVIDED_CLASSES` constant to another file to avoid import problems.